plugins/inputs/qbittorrent/README.md

```diff
@@ -0,0 +1,104 @@
+# QBittorrent Input Plugin
+
+The qbittorrent plugin will get torrents status.
+
+Compatible qbittorrent API versions must be higher than 2.9.2.
+Versions lower than 2.9.2 and higher than 2.1.1 are also compatible
+in principle but have not been tested.
+
+## Global configuration options
+
+## Configuration
+
+```toml @sample.conf
+# Read QBittorrent status information
+[[inputs.qbittorrent]]
+  ## Url of QBittorrent server
+  # Tls can be:
+  # url = "https://127.0.0.1:8080"
+  url = "http://127.0.0.1:8080"
+
+  ## Credentials for QBittorrent
+  # username = "admin"
+  # password = "admin"
+```
+
+## Metrics
+
+- server_state
+  - all_time_download
+  - all_time_upload
+  - average_time_queue
+  - connection_status
+  - dht_nodes
+  - dl_info_data
+  - dl_info_speed
+  - dl_rate_limit
+  - free_space_on_disk
+  - global_ratio
+  - queued_io_jobs
+  - queueing
+  - read_cache_hits
+  - read_cache_overload
+  - refresh_interval
+  - total_buffers_size
+  - total_peer_connections
+  - total_queued_size
+  - total_wasted_session
+  - up_info_data
+  - up_info_speed
+  - up_rate_limit
+  - use_alt_speed_limits
+  - use_subcategories
+  - write_cache_overload
+
+- torrents
+  - added_on
+  - amount_left
+  - availability
+  - completed
+  - completion_on
+  - download_limit
+  - download_speed
+  - downloaded
+  - downloaded_session
+  - eta
+  - inactive_seeding_time_limit
+  - last_activity
+  - max_inactive_seeding_time
+  - max_ratio
+  - max_seeding_time
+  - num_complete
+  - num_incomplete
+  - num_leechs
+  - num_seeds
+  - priority
+  - progress
+  - ratio
+  - ratio_limit
+  - seeding_time
+  - seeding_time_limit
+  - seen_complete
+  - size
+  - time_active
+  - total_size
+  - trackers_count
+  - up_limit
+  - uploaded
+  - uploaded_session
+  - upspeed
+
+- tags
+  - count
+
+- category
+  - count
```

Comment:
Does these 2 need to be separate metrics? Can't they be as fields `tags` and `categories` on the `server_state` metric?

Reply:
The calculated data type of `tags` and `category` is telegraf.Counter [code link](https://github.com/RWDai/telegraf/blob/dfd916d25c3eb509daaf6d894a972343cc2bdbf9/plugins/inputs/qbittorrent/qbittorrentdto.go#L519), and the other directly collected data type is telegraf.Gauge, so I separated them. I'm not sure about these two concepts, should I still merge them into server_state?

Reply:
I would not care about them, it is a bad interpretation for prometheus compatibility..

So yeah, please merge.